devops/providers/azure-devops/templates/infrastructure/README.md

```diff
@@ -183,6 +183,7 @@ There should be one Azure Unified pipeline for each individual application. This
 | Template location | Pipeline | The relative path of the terraform template | None |
 | Prefix name | Pipeline | the prefix name for the provisioned azure resources | None |
 | Location | Pipeline | The data center location | None |
+| IP Whitelist Ranges | Pipeline | The Azure DevOps build agent whitelisted IP ranges to configure for the azure resources that need to be accessible for the pipeline tasks | None |
 | ASE resource name | Pipeline | The resource name for the azure service environment to use for the service plan deployment | None |
 | ASE resource group name | Pipeline | The resource name for the azure service environment to use for the service plan deployment | None |
 | Container Image Build Git Source URL | Pipeline | The URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') containing the docker build manifests | https://github.com/erikschlegel/echo-server.git |
```

Comment:
I think this will be checked out automatically since the build is triggered off of commits to this repo. No need to supply as a URL